src/reducers/transactionReducer.js

```diff
@@ -399,6 +402,8 @@ export default (state = initialState(), action) => {
       return validateTransaction(updateOutputAddress(state, action));
     case SET_OUTPUT_AMOUNT:
       return validateTransaction(updateOutputAmount(state, action));
+    case SET_UNSIGNED_PSBT:
+      return updateState(state, { unsignedPSBT: action.value });
```

Comment:
future TODO but I suspect this probably didn't need to be in the redux store at all. Probably something we need to start moving away from eventually in Caravan.